dev/io.openliberty.io.smallrye.openapi41.core/resources/io/smallrye/openapi/api/util/io.smallrye.openapi.api.util.VersionUtil.properties

```
@@ -0,0 +1 @@
+microprofile.openapi.version=4.0
```

```suggestion
microprofile.openapi.version=4.1
```
Should it be 4.1 or you won't change that until you update the bundle to actually have 4.1 content in it?